src/steamship_langchain/__init__.py

```diff
@@ -0,0 +1,5 @@
+from langchain import LLMChain, PromptTemplate
+
+from steamship_langchain.llms.openai import OpenAI
+
+__all__ = ["LLMChain", "PromptTemplate", "OpenAI"]
```

Comment:
curious: what is the intent here (echo-ing these classes all inside the `steamship_langchain` namespace)?

Is this part of a larger vision for one-click deployment?

Just want to be sure I track intent.

Reply:
I'm exploring the world where we import most of LangChain's functionality and expose it under steamship_langchain. 

Doing so would allow people to just swap `from langchain` with `from steamship_langchain` and be done with it. 

That said though, after having explored the different use-cases I think that's a bad idea to pursue. Most Steamship-specific implementations will require you to swap one class for another. For example, users will have to swap `FAISS` for `SteamshipVectorStore`. In these cases just swapping langchain for steamship_langchain will not do the job.